### PR TITLE
Move contents of the conference page to the events page

### LINKED ISSUE #413: Combine Conferences and Events to one page
Currently we have the events page with one event (Coffee and Code) and the conferences page that is dynamically built using `_conferences/__main__.py`.

The plan is to combine the two pages to be `events.html`

```mermaid
graph TD
    A[Webpage] --> B[Header]
    B --> C[BPD Events]
    C--> D[Upcomming Conferences]
    D --> E[Footer]
```

### PR DESCRIPTION
**Pull request by TimOsahenru**
- [x] Conference page removed on all languages

- [x] Events page updated to have an *Upcoming conferences* section

### CODE REVIEW

**Comment by TimOsahenru** [comment]
@kjaymiller I ran the command `pre-commit run --all-files` before pushing not sure why I still have tests

File: _articles/sw/events.md
Comment (kjaymiller) [inline _articles/sw/events.md:1]: It seems like the swahili page is not being dynamically being generated. I would prefer that we include `include conferences.html` like what was done in the other one.

File: _config.yml
Comment (kjaymiller) [inline _config.yml:1]: instead of commenting out you can remove these

File: _data/locales/en.yml
Comment (kjaymiller) [inline _data/locales/en.yml:1]: don't comment, remove

File: _data/locales/es.yml
Comment (kjaymiller) [inline _data/locales/es.yml:1]: remove

File: _data/locales/sw.yml
Comment (kjaymiller) [inline _data/locales/sw.yml:1]: remove

**Review by kjaymiller — CHANGES_REQUESTED** [review]
Great work on getting this.

Biggest comment is you can remove the contents. leaving them in the comments is not necessary.

**Comment by TimOsahenru** [comment]
@kjaymiller made the necessary changes, but I still can't these linting tests to pass. Also, noticed the event page for Hausa translation returns a 404 page, created an issue for it https://github.com/BlackPythonDevs/blackpythondevs.github.io/issues/415

**Pull request merged**